### PR TITLE
Add justification for trove classifier based CUDA version check

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/thoth-application/issues/1850

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: _j/trove_cuda.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: No CUDA specific trove classifier matching CUDA version used
+---
+
+Package maintainer did not state CUDA version used as a compatible version.
```
Comment (KPostOffice) [inline _j/trove_cuda.md:5]:
```suggestion
The CUDA version used is not listed as a compatible version for this package by the package maintainer.
```

**Review by KPostOffice — COMMENTED** [review]
One small comment

File: _j/trove_cuda.md (hunk 1 above)
Comment (fridex) [reply _j/trove_cuda.md]: Thanks, applied 👍🏻

**Pull request merged**